product-service/src/functions/getProductsList/handler.ts

```diff
@@ -5,12 +5,18 @@ import { getProducts } from '@libs/data-service';
 
 export const main: APIGatewayProxyEventHandler = async () => {
   try {
+    console.log('Request: products list');
```

Comment:
**Suggestion**: If you have middleware you could put logger there and remove all redundant logs.

Reply:
yes I was thinking about adding some middleware logger would be much better if it's an enterprise project, but the task says "do console.log" and it's quite enough for seeing the logs in Cloudfront. but it'll require some more time to do it better :(

Reply:
decided to leave it for the future freed up myself, sorry. I'm fine if you consider these logs as 0 points though :)